src/js/models/panel_model.js

```diff
@@ -279,47 +295,107 @@
                 isNegative = (deltaT < 0),
                 text = "", h, m, s;
             deltaT = Math.abs(deltaT);
+            h = parseInt(deltaT / 3600);
+            m = parseInt(deltaT / 60);
+            s = pad(Math.round(deltaT % 60));
             if (format === "index") {
                 isNegative = false;
                 text = "" + (theT + 1);
-            } else if (format === "milliseconds") {
+            } else if (['milliseconds', 'ms'].includes(format)) {
                 text = Math.round(deltaT*1000) + " ms";
-            } else if (format === "secs") {
+            } else if (['seconds', 'secs', 's'].includes(format)) {
                 text = Math.round(deltaT) + " s";
-            } else if (format === "mins:secs") {
-                m = parseInt(deltaT / 60);
-                s = pad(Math.round(deltaT % 60));
-                text = m + ":" + s;
-            } else if (format === "mins") {
+            } else if (['minutes', 'mins', 'm'].includes(format)) {
                 text = Math.round(deltaT / 60) + " mins";
-            } else if (format === "hrs:mins") {
-                h = parseInt(deltaT / 3600);
-                m = pad(Math.round((deltaT % 3600) / 60));
+            } else if (["mins:secs", "m:s"].includes(format)) {
+                text = m + ":" + s;
+            } else if (["hrs:mins", "h:m"].includes(format)) {
                 text = h + ":" + m;
-            } else if (format === "hrs:mins:secs") {
-                h = parseInt(deltaT / 3600);
-                m = pad(parseInt((deltaT % 3600) / 60));
-                s = pad(Math.round(deltaT % 60));
+            } else if (["hrs:mins:secs", "h:m:s"].includes(format)) {
```

Comment:
You've lost the `pad()` on the mins here and on the mins for `hrs:mins` above. E.g. `0:00:00` now reads as `0:0:00`. NB: for `mins:secs` you don't want to `pad(m)`.

Reply:
Sorry, I thought I could handle it all at once, but I missed that pad part. I reverted my changes